src/polynomial.h

```diff
@@ -387,25 +387,6 @@ void Polynomial<T>::_extended_gcd(
     }
 }
 
-/** Derivative of a polynomial
```

Comment:
commit message says: "improve derivative", commit message should also explain how